MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -977,7 +977,7 @@ public void OnRefreshComplete(BaseItem item)
 
         public void OnRefreshProgress(BaseItem item, double progress)
         {
-            //_logger.LogInformation("OnRefreshProgress {0} {1}", item.Id.ToString("N"), progress);
+            //_logger.LogInformation("OnRefreshProgress {0} {1}", item.Id.ToString("N", CultureInfo.InvariantCulture), progress);
```

Comment:
This commented line should probably either be put behind a debug compiler directive or deleted.

Reply:
I agree, the three `LogInformation` instances should get changed to `LogDebug` and enabled. The exception below should also get changed to either warn or error so we can start getting some logging for that issue.

Reply:
100% yes. We need more debug logs to catch issues anyways, no sense commenring out valid logging when we have a lower prio to throw it under.

Reply:
Yeah, but that's out of the scope of this PR imo